MANIFEST.in

```diff
@@ -1,13 +1,8 @@
 graft src/pudl/package_data
```

Comment:
If we remove this `graft` line, the package_data is still included because we defined it in `pyproject.toml`.